pkg/controller/config/config_controller_test.go

```diff
@@ -417,8 +421,16 @@ func TestConfig_CacheContents(t *testing.T) {
 
 	// Create the Config object and expect the Reconcile to be created
 	ctx = context.Background()
-	err = c.Create(ctx, instance)
-	g.Expect(err).NotTo(gomega.HaveOccurred())
+
+	createInstance := func() client.Object {
+		return configFor([]schema.GroupVersionKind{nsGVK, configMapGVK})
```

Comment:
Isn't this redefining `instance`?  How do these declarations compare?

Reply:
We need a fresh copy of Instance each time we send one to a call to `client.Create`/`client.Delete` as they modify instance.

Reply:
Perhaps use `instance.DeepCopy()` then?  So we know they are the same?

Reply:
Ok. I don't see a distinction in the approaches. All that matters is that for each call to a `client.Client` method, we have a new copy.